apps/prairielearn/elements/pl-code/pl-code.py

```diff
@@ -112,6 +126,40 @@ def get_lexer_by_name(name: str) -> Optional[pygments.lexer.Lexer]:
             return None
 
 
+# Computing this is relatively expensive, so we'll do it once here and reuse it.
```

Comment:
Nit: Why not set this as a constant? Just `COLOR_TOKENS = color_tokens(ANSI_COLORS, ANSI_COLORS)`. I guess either style is fine, but looks a little funny to my eye.

Reply:
It's written this way because this value is (moderately) expensive to compute and we only need it during `render()`, so we avoid creating it entirely until it's actually needed.